lib/core/src/display/rendering/transform.rs

```diff
@@ -57,6 +57,9 @@ impl Transform {
         &self.translation
     }
 
+    /// Gets mutable Transform's translation.
+    pub fn translation_mut(&mut self) -> &mut Vector3<f32> { &mut self.translation }
+
```

Comment:
Let's not write functions in single line until you define multiple glued functions this way please :)

Reply:
@wdanilo what do you mean by `multiple glued functions`?